example/package.json

```diff
@@ -12,6 +12,8 @@
   "dependencies": {
     "@react-navigation/native": "^6.0.11",
     "@react-navigation/native-stack": "^6.7.0",
+    "@shopify/flash-list": "^1.6.4",
```

Comment:
@prince-d-simform We should mention these dependencies in readme installation steps